source/features/download-folder-button.tsx

```diff
@@ -1,25 +1,44 @@
+import React from 'dom-chef';
+import {$, expectElement} from 'select-dom';
 import * as pageDetect from 'github-url-detection';
+import DownloadIcon from 'octicons-plain-react/Download';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
+import replaceElementTypeInPlace from '../helpers/recreate-element.js';
 
-function add({parentElement: deleteDirectoryItem}: HTMLAnchorElement): void {
-	const item = deleteDirectoryItem!.cloneNode(true);
-	const link = item.firstElementChild as HTMLAnchorElement;
+function add(menu: HTMLUListElement): void {
 	const downloadUrl = new URL('https://download-directory.github.io/');
 	downloadUrl.searchParams.set('url', location.href);
+
+	const item = menu.firstElementChild!.cloneNode(true);
+	item.role = 'none';
+	item.removeAttribute('tabindex');
+	item.removeAttribute('id');
+	item.removeAttribute('aria-keyshortcuts');
+	item.removeAttribute('aria-labelledby');
```

Comment:
I had to fight a bit to keep it consistent and accessible. The result now is that it works correctly in both views, except that on permalinks the outline is smaller. 🤷‍♂️ Not enough of a problem to care.

This is me navigating both menus with arrows or <kbd>c</kbd>

![permalink](https://github.com/user-attachments/assets/336b01f9-baea-45cd-b617-d046b8f0dc29)

 
![regular](https://github.com/user-attachments/assets/baaeceb4-ca7b-46b8-a472-59cf2638f8f6)